spec/features/user_sees_greeting_spec.rb

```diff
@@ -0,0 +1,18 @@
+require_relative '../spec_helper.rb'
+
+describe "when a user creates a horse" do
+  it "they see a form" do
+    j = Jockey.create(name: "Garfield")
+    new_horse = "/jockeys/#{j.id}/horses/new"
+    puts new_horse
+    visit(new_horse)
+    fill_in("horse[name]", with: "Sunshine")
+    fill_in("horse[age]", with: 22)
+    fill_in("horse[total_winnings]", with: 5000)
+    click_on("Submit")
+
```

Comment:
When you're navigated to a new page via link, button or form, really any other way than using the explicit #visit method, make an assertion about the path of the page you end up on. For example:

```ruby
expect(current_path).to eq("/horses/#{Horse.last.id}")
```